admin/app/components/solidus_admin/orders/cart/component.js

```diff
@@ -0,0 +1,129 @@
+import { Controller } from "@hotwired/stimulus"
+import { useClickOutside, useDebounce } from "stimulus-use"
+
+const LOADING = "Loading..."
+const INITIAL = "Type to search"
+const EMPTY = "No results"
```

Comment:
nit: How about passing these messages as I18n data values?

Reply:
Great catch. Would even call this mandatory and not a nit pick imo ;)